tests/unit/docker/test_hub_build_level.py

```diff
@@ -40,18 +40,18 @@ def test_hub_build_level_pass(monkeypatch, test_workspace, docker_image):
     expected_failed_levels = []
 
     _, failed_levels = HubIO(args)._test_build(docker_image, BuildTestLevel.EXECUTOR,
-                                               os.path.join(cur_dir, 'yaml/test-joint.yml'), 60, True,
+                                               os.path.join(cur_dir, 'yaml/test-joint.yml'), 60000, True,
                                                JinaLogger('unittest'))
 
     assert expected_failed_levels == failed_levels
 
 
 def test_hub_build_level_fail(monkeypatch, test_workspace, docker_image):
     args = set_hub_build_parser().parse_args(['path/hub-mwu', '--push', '--host-info', '--test-level', 'FLOW'])
-    expected_failed_levels = [BuildTestLevel.POD_NONDOCKER, BuildTestLevel.POD_DOCKER, BuildTestLevel.FLOW]
+    expected_failed_levels = [BuildTestLevel.POD_DOCKER, BuildTestLevel.FLOW]
```

Comment:
Can someone verify, that this change makes sense? @rutujasurve94 @JoanFM
I am not sure about the consequences of this change and only did it, since the test expected it.

Reply:
Yes I do not see why it should Fail in the other one